packages/components/app/styles/components/table.scss

```diff
@@ -103,6 +131,36 @@
   background-color: var(--token-color-surface-primary);
 
   .hds-table__tr {
-    border-bottom: 1px solid var(--token-color-border-primary);
+    td {
+      border-top: none;
+      border-right: $hds-table-border-width solid $hds-table-border-color;
+      border-bottom: $hds-table-border-width solid $hds-table-border-color;
+      border-left: none;
```

Comment:
I see here you're using a different approach: you're applying the color to the `hds-table__tbody` and not to the `TD` (compared to the table head). maybe use the same approach you've used above, also here? (apply the background color and the text color to the `TD` element)?